### PR TITLE
MAT-6156 resolve getExpectedValue exception with empty string

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE FHIR SERVICE

Jira Ticket: [MAT-6156](https://jira.cms.gov/browse/MAT-6156)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/madiefhirservice/services/TestCaseBundleService.java [hunk 1]
```diff
@@ -204,7 +206,7 @@ private List<MeasureReport.MeasureReportGroupComponent> buildMeasureReportGroupC
    * @return an equivalent integer
    */
   private int getExpectedValue(Object expectedValue) {
-    if (expectedValue == null) {
+    if (expectedValue == null || StringUtils.isBlank(expectedValue.toString())) {
```
Comment (RohitKandimalla) [inline src/main/java/gov/cms/madie/madiefhirservice/services/TestCaseBundleService.java:209]: StringUtils.isBlank will also look for null checks, so I would say our null check is redundant, We can safely remove it.

Comment (sb-cecilialiu) [reply src/main/java/gov/cms/madie/madiefhirservice/services/TestCaseBundleService.java:209]: Thanks Rohit! If expectedValue is a String type, then we can remove the null check, but I needed to convert the object to String, there might be an exception there? @RohitKandimalla

Comment (RohitKandimalla) [reply src/main/java/gov/cms/madie/madiefhirservice/services/TestCaseBundleService.java:209]: aah ok, I missed that.

**Review by RohitKandimalla — APPROVED** [review]
